packages/js/components/changelog/dev-move-report-components

```diff
@@ -0,0 +1,4 @@
+Significance: patch
```

Comment:
From the perspective of `@woocommerce/components`, would it be a bit closer to the minor level?